lib/transactional.pm

```diff
@@ -39,10 +40,13 @@ sub process_reboot {
         microos_reboot $trigger;
     } else {
         power_action('reboot', observe => !$trigger, keepconsole => 1);
-
-        # Replace by wait_boot if possible
-        assert_screen 'grub2', 100;
-        send_key 'ret';
+        if (check_var('ARCH', 's390x')) {
+            reconnect_mgmt_console(timeout => 500);
```

Comment:
Have you tried `boot/reconnect_mgmt_console.pm`? I wonder if the module works why do we need to mess with this file. if the `boot/reconnect_mgmt_console.pm` module doesnt work then we need to adjust it. WDYT?

Reply:
No, as you can see in the test run, we call this function multiple times, for each reboot.